### ISSUE TITLE
TypeScript module resolution in fixtures

### ISSUE THREAD

**Issue by harveyappleton** [jeanfortheweb/typeorm-fixture-builder#45]
Hey @jeanfortheweb, 

Firstly thanks so much for an awesome library, top notch work!! 💯 

Having an issue where, in my project, I have set in **tsconfig.json** the property: `"baseUrl": "./src"`, means that typeorm-fixture-builder is struggling to resolve entities.

My fixture may look like this for example:

```typescript
import { payeeFixture } from 'entities/Payee.entity';

export const payees = [payeeFixture(), payeeFixture(), payeeFixture()];
```
_where entities is a folder that could be resolved by eg `../../entities/Payee.entity`_

But console ouput when I try to install fixtures:
```
$ /Users/harvey.appleton/Documents/***/***/node_modules/.bin/fixtures install 'src/fixtures/demo/**/*.bundle.ts'
✔ Database connected
ℹ Database reset skipped
Error: Cannot find module 'entities/Payee.entity'
Require stack:
- /Users/harvey.appleton/Documents/***/***/src/fixtures/demo/payee.bundle.ts
- /Users/harvey.appleton/Documents/***/***/node_modules/typeorm-fixture-builder/lib/cjs/cli/task/collect.js
- /Users/harvey.appleton/Documents/***/***/node_modules/typeorm-fixture-builder/lib/cjs/cli/executor.js
- /Users/harvey.appleton/Documents/***/***/node_modules/typeorm-fixture-builder/lib/cjs/cli.js
- /Users/harvey.appleton/Documents/***/***/node_modules/typeorm-fixture-builder/bin/cli.js
error Command failed with exit code 1.
```
(I've *** out some folder names).

Any advice/ideas as to how I could keep my non-relative modules? I know the work-around would be to change the imports to be relative, but wondering if there's a better way we could fix it.

Cheers!

**Comment by jeanfortheweb** [comment]
Hey harvey, if I get this right, I guess, the cli does not know about the custom paths. I feel like https://www.npmjs.com/package/tsconfig-paths could help there, because it makes your baseURL/aliases known to the node/ts-node process. Maybe, we could even load this by default in the CLI. 

I'll have a look at it asap, but in the meantime, you could try to use it?

**Comment by harveyappleton** [comment]
Hey @jeanfortheweb ! thanks for the recommendation.
The way I managed to make it work was this in my **package.json:**

```
    "fixtures:demo": "node --require ./node_modules/tsconfig-paths/register ./node_modules/typeorm-fixture-builder/bin/cli.js -- install src/fixtures/demo/**/*.bundle.ts",
```

Which basically makes node require `tsconfig-paths/register`, which resolves to the **tsconfig.json** in my project root. It then calls the **cli.js** file from the `typeorm-fixture-builder` package in my **node_modules** folder.

Like you said, perhaps it would be good to include `tsconfig-paths/register` by default in the cli file? 🤔

**Comment by jeanfortheweb** [comment]
Yes I think I'll make a patch for that solution, since we already using another one!

**Comment by jeanfortheweb** [comment]
Just published 2.2.2 which includes tsconfig-paths by default. Please let me know if it's working as intended :)

**Comment by harveyappleton** [comment]
Works great, thanks @jeanfortheweb ! 🎉
